src/components/v5/common/ActionSidebar/partials/MultiSigSidebar/partials/Signees/Signees.tsx

```diff
@@ -10,16 +13,64 @@ interface SigneesProps {
   signees: MultiSigUserSignature[];
 }
 
-const Signees: FC<SigneesProps> = ({ signees }) => (
-  <div className="flex flex-col">
-    {signees.map((signature) => (
-      <div className="flex w-full flex-row justify-between">
-        <span>{signature.userAddress.slice(0, 10)}...</span>
-        <span>{signature.vote}</span>
-      </div>
-    ))}
-  </div>
-);
+const renderVotes = (vote: MultiSigVote) => {
+  switch (vote) {
+    case MultiSigVote.Approve:
+      return (
+        <div className="flex items-center gap-1">
+          <Check size={14} />
+          <span>Approved</span>
+        </div>
+      );
+    case MultiSigVote.Reject:
+      return (
+        <div className="flex items-center gap-1">
+          <X size={14} />
+          <span>Opposed</span>
+        </div>
+      );
+    default:
+      return <div className="text-gray-300">Awaiting</div>;
+  }
+};
+
+const Signees: FC<SigneesProps> = ({ signees }) => {
+  const [loadedSignees, setLoadedSignees] = useState(5);
+
+  const loadMore = () => {
+    setLoadedSignees((prev) => prev + 5);
+  };
+
+  return (
+    <div>
+      <ul className="flex flex-col">
+        {signees.slice(0, loadedSignees).map((signature) => (
+          <li
+            className="mb-3 flex w-full flex-row justify-between last:mb-0"
+            key={signature.userAddress}
+          >
+            <UserPopover
+              size={20}
+              walletAddress={signature.userAddress}
+              popperOptions={{ placement: 'bottom-end' }}
+              withVerifiedBadge={false}
+            />
+            <div className="text-sm">{renderVotes(signature.vote)}</div>
+          </li>
+        ))}
+      </ul>
+      {signees.length > 5 && loadedSignees < signees.length && (
```

Comment:
nitpicky, but I think we can remove the first condition and always check if we have less loaded than the total length